chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedReducer.kt

```diff
@@ -9,5 +9,7 @@ internal val joinedRoomsReceivedReducer =
 
         val joinedRooms = action.rooms.map { it.id to it }.toMap()
         val joinedRoomsState = JoinedRoomsState(joinedRooms, action.unreadCounts)
-        state.with(joinedRoomsState, state.auxiliaryState.with(action))
-}
+        state.with(
+            joinedRoomsState = joinedRoomsState
+        )
```

Comment:
minor but if it fits 1 line it's usually more legible to not make it 3 lines (as it used to be here), also named param here is just noise

Reply:
It matches all the other invocations in other files, which makes it easier to read. I added the names parameter as we expect more in the (very near) future, right?

Reply:
Improved.